pattern-2/tile/tile.yml

```diff
@@ -31,12 +31,25 @@ forms:
   label: Carbon DB connection information
   description: WSO2 API Manager - Carbon Database connection information
   properties:
-  - name: carbon_db_hostname
+  - name: carbon_jdbc_url
     type: string
-    label: Hostname
-  - name: carbon_db_name
+    label: JDBC URL
+  - name: carbon_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+    - name: postgres
```

Comment:
We do not support this at the moment. Shall we remove this for the moment?

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c